src/Annotations/Serializer/CommonMark/HtmlPurifierRenderer.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace eLife\Annotations\Serializer\CommonMark;
+
+use HTMLPurifier;
+use League\CommonMark\Block\Element\AbstractBlock;
+use League\CommonMark\ElementRendererInterface;
+use League\CommonMark\Environment;
+use League\CommonMark\Inline\Element\AbstractInline;
+use RuntimeException;
+
+class HtmlPurifierRenderer implements ElementRendererInterface
+{
+    private $htmlPurifier;
+    protected $environment;
+
+    /**
+     * @param Environment $environment
+     */
+    public function __construct(Environment $environment, HTMLPurifier $htmlPurifier)
```

Comment:
I meant this should compose the original `HtmlRender` rather than duplicate everything...